Bio/SearchIO/BlastIO/blast_xml.py

```diff
@@ -384,7 +384,7 @@ def _parse_qresult(self):
                 yield qresult
 
     def _parse_hit(self, root_hit_elem, query_id):
-        """Generator that transforms Iteration_hits XML elements into Hit objects.
+        """Transform Iteration_hits XML elements into Hit objects (PRIVATE).
```

Comment:
I'd like the docstring to indicate this is a generator (iterator), not sure how best to rephrase this though.

Reply:
What about `Yield a generator object that transforms Iteration_hits XML elements into Hit objects.`?